x-pack/plugins/observability_solution/observability_shared/common/color_palette.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { difference, first, values } from 'lodash';
+import { euiPaletteColorBlind } from '@elastic/eui';
+
+export enum Color {
+  color0 = 'color0',
+  color1 = 'color1',
+  color2 = 'color2',
+  color3 = 'color3',
+  color4 = 'color4',
+  color5 = 'color5',
+  color6 = 'color6',
+  color7 = 'color7',
+  color8 = 'color8',
+  color9 = 'color9',
+}
```

Comment:
You could probably save some lines of code if you use a type instead:
```
type Color = 'color0' | 'color1' | 'color2';
```

Reply:
@cauemarcondes I applied the suggested change.